packages/api/src/command/medical/document/document-webhook.ts

```diff
@@ -67,8 +67,17 @@ export const processPatientDocumentRequest = async (
       ],
     };
 
+    const metadata =
+      whType === "medical.document-download" || whType === "medical.document-conversion"
+        ? patient.data.cxDocumentRequestMetadata
+        : whType === "medical.consolidated-data"
+        ? patient.data.cxConsolidatedRequestMetadata
+        : whType === "medical.document-bulk-download-urls"
+        ? patient.data.cxDownloadRequestMetadata
+        : undefined;
```

Comment:
massive chained ternary operators are a code smell since they're hard to grok at a glance - can we update this to an if/else chain?

Reply:
yah I was just trying to avoid using let, but can use if else instead

Reply:
Can create a local function to return the value, so you can still use `const`. :)